xclim/subset.py

```diff
@@ -377,18 +342,83 @@ def create_mask(
     >>> region_names = xr.DataArray(polys.id, dims=('regions',)))
     >>> ds = ds.assign_coords(regions_names=region_names)
     """
-    wgs84 = CRS("+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs")
-    # poly.crs = wgs84
+    _check_has_overlaps(polygons=poly)
```

Comment:
I would make this option I think... It can be super long with a shape that has lots of polyogns